Makefile

```diff
@@ -1,81 +1,17 @@
-GITBOOK = node_modules/.bin/gitbook
-JSDOC = node_modules/.bin/jsdoc
-LESS = node_modules/.bin/lessc
-
-JSDOC_FILES := $(shell find jsdoc -type f | sort)
-LESS_FILES := $(shell find less -name '*.less' | sort)
-
-
 .PHONY: all
-all: \
-	check-version \
-	$(VERSION)/tmp/README.md \
-	$(VERSION)/tmp/package.json \
-	$(VERSION)/docs/dist/ramda.js \
-	$(VERSION)/docs/index.html \
-	$(VERSION)/docs/main.js \
-	$(VERSION)/index.html \
-	$(VERSION)/fonts/glyphicons-halflings-regular.eot \
-	$(VERSION)/fonts/glyphicons-halflings-regular.svg \
-	$(VERSION)/fonts/glyphicons-halflings-regular.ttf \
-	$(VERSION)/fonts/glyphicons-halflings-regular.woff \
-	$(VERSION)/fonts/glyphicons-halflings-regular.woff2 \
-	$(VERSION)/style.css \
```

Comment:
Couldn't find a reason to keep all the documentations for all versions. We can use Git for that I think. Removing that allowed me to greatly simplify the entire Makefile and make the documentation generation process a bit simpler to follow.

Reply:
Yeah, they were a good idea at the start, but we've never really used them.